workflow/rocoto/gfs_tasks.py

```diff
@@ -18,6 +18,15 @@ def _is_this_a_gdas_task(run, task_name):
     # Specific Tasks begin here
     def stage_ic(self):
 
+        dependencies = None
+        if self.options['do_fetch_hpss'] or self.options['do_fetch_local']:
+            deps = []
+            dep_dict = {
+                'type': 'task', 'name': f'fetch',
```

Comment:
The task name should also have the `RUN` in it:

```suggestion
                'type': 'task', 'name': f'{self.run}_fetch',
```

Reply:
Done